jmix-translations/content/io/jmix/notificationsflowui/messages_de.properties

```diff
@@ -27,9 +27,10 @@ io.jmix.notificationsflowui.view.createnotification/channelsField.label=Kanäle
 io.jmix.notificationsflowui.view.createnotification/plainTextBodyField.label=Nachricht
 
 io.jmix.notificationsflowui.view.receivedinappnotifications/receivedInAppNotificationsDialog.title=In-App-Benachrichtigungen
-io.jmix.notificationsflowui.view.receivedinappnotifications/markAsReadUnreadBtn.text.markAsRead=Als gelesen markieren
-io.jmix.notificationsflowui.view.receivedinappnotifications/markAsReadUnreadBtn.text.markAsUnread=Als ungelesen markieren
-io.jmix.notificationsflowui.view.receivedinappnotifications/markAllAsReadBtn.text=alle als gelesen markieren
+io.jmix.notificationsflowui.view.receivedinappnotifications/openAction.text=Öffnen
```

Comment:
Translation of 'Open' should be delivered separately

Reply:
removed